nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/package.json

```diff
@@ -6,7 +6,7 @@
         "start": "ng serve",
         "build": "ng build",
         "watch": "ng build --watch --configuration development",
-        "test": "ng test",
+        "test": "ng test --karma-config=karma.conf.js",
```

Comment:
@mcgilman,
Currently, running `npm run test` just hangs and doesn't complete. We have a few options:
* update this script to: `"test": "ng test --karma-config=karma.conf.js --watch=false",`
* add another script: `"test-no-watch": "ng test --karma-config=karma.conf.js --watch=false",`
* update the karma.conf.js to specify this is a `singleRun` test by adding `singleRun: true`
* add another karma configuration for the singleRun of tests.